C++/fizzbuzz.cpp

```diff
@@ -3,44 +3,39 @@ Problem : a) print "FizzBuzz" ,if N (user input) is divisile to 3 and 5,
 		  b) print "Fizz" ,if N (user input) is divisile to 3 , 
 		  c) print "Buzz" ,if N (user input) is divisile to 5
 */
+// There is no need of "for" loop to print output as you mentioned above.
+//If it is not the way you want you can mention it again.
+
```

Comment:
Hi @MaxJaycee looping is required as you need to print the pattern of `1, 2, Fizz, 4, Buzz....`

Reply:
ok i will try for it again.